apps/emqx_cluster_link/src/emqx_cluster_link.erl

```diff
@@ -245,6 +246,7 @@ actor_init(
     end.
 
 actor_init_ack(#{actor := Actor}, Res, MsgIn) ->
+    ?tp("clink_actor_init_ack_enter", #{}),
```

Comment:
Nit. Seems unused, worth to keep?

Reply:
oops :see_no_evil:  , I was going to use this with `?force_ordering` at some point and forgot it here.  removed.